screens/home/HomeScreen.js

```diff
@@ -394,13 +393,18 @@ class HomeScreen extends PureComponent {
             // async loop
             for (const img of this.props.images) {
                 const isgeotagged = isGeotagged(img);
-                const myIndex = this.props.images.indexOf(img);
                 if (
                     img.type !== 'WEB' &&
                     img.tags &&
                     Object.keys(img.tags).length > 0 &&
                     isgeotagged
                 ) {
+                    const date =
+                        img.type === 'GALLERY'
+                            ? moment
+                                  .unix(img.date)
+                                  .format('YYYY:MM:DD HH:mm:ss')
+                            : img.date;
```

Comment:
why don't we do this when the image is selected?

Reply:
We can do that when we select image from Galley... better to do that there, I will move this

Reply:
Current backend can parse epoch timestamp.. So sending epoch now, and removed this date formatting on frontend e619afa4739ede4b26d03a8717c2de93bdf5610d